src/bin/bat/clap_app.rs

```diff
@@ -257,6 +257,15 @@ pub fn build_app(interactive_output: bool) -> ClapApp<'static, 'static> {
                      an interactive terminal is detected. Possible values: *auto*, never, always.",
                 ),
         )
+        .arg(
+            Arg::with_name("always-decorations")
```

Comment:
With the extended functionality, I would rename this to `--force-colorization` maybe?